performance.rst

```diff
@@ -42,6 +43,25 @@ every request. There are some `byte code caches`_ available, but as of PHP
 5.5, PHP comes with `OPcache`_ built-in. For older versions, the most widely
 used byte code cache is `APC`_.
 
+.. _performance-use-preloading:
+
+Use the OPcache class preloading
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 4.4
+
+    The feature that generates the preloading file was introduced in Symfony 4.4.
+
+Starting from PHP 7.4, OPcache can compile and load classes at start-up and
+make them available to all requests until the server is shut down, improving
+performance significantly. Symfony generates automatically the file with the
+list of classes to preload:
+
+.. code-block:: ini
+
+    ; php.ini
+    opcache.preload=/path/to/project/var/cache/prod/srcApp_KernelProdContainer.preload.php
```

Comment:
Maybe add a note that the « prod » is dynamic and dependant of the app env
Because on a preprod server, it can be also usefull to use this feature :)